torchgen/gen_backend_stubs.py

```diff
@@ -265,7 +265,10 @@ def error_on_missing_kernels(
             native_f
         )
 
-    kernel_defn_regex = rf"{class_name}::([\w\d]*)\([^\)]*\)\s*{{"
+    # This just looks for lines containing "foo(", and assumes that the kernel foo has been implemented.
+    # It might cause false negatives (we won't catch all cases), but that's ok - if we catch a missing kernel
+    # here, then we get a nicer error message. If we miss it, you get a linker error.
+    kernel_defn_regex = r"([\w\d]*)\("
```

Comment:
```suggestion
    # This just looks for lines containing "foo(", and assumes that the kernel foo has been implemented.
    # It might cause false negatives (we won't catch all cases), but that's ok - if we catch a missing kernel
    # here, then we get a nicer error message. If we miss it, you get a linker error.
    kernel_defn_regex = r"f"{class_name}::\s*([\w\d]*)\("
```

Reply:
you're right, thanks.

I made that change originally because our auto-formatter was taking a kernel with a long signature like ` LazyNativeFunctions::native_group_norm` and moving the `native_group_norm(` bit onto a new line, but I was able to fix things up so it's all on the same line. So I can go with what you put (updating soon!)